### PR TITLE
Use payment loading flag

### PR DESCRIPTION
**Pull request by harshvitra**
Solved multiple donations when using Gifts + Native pay

### CODE REVIEW

File: src/features/donations/components/PaymentRequestForm.tsx [hunk 1]
```diff
@@ -145,12 +146,14 @@ export const PaymentRequestCustomButton = ({
   }, [paymentRequest]);
 
   useEffect(() => {
-    if (paymentRequest) {
+    if (paymentRequest && !paymentLoading) {
```
Comment (norbertschuler) [inline src/features/donations/components/PaymentRequestForm.tsx:149]: Do you think this change should also be done in https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/features/legacyDonations/components/PaymentRequestForm.tsx which is mostly the same code?

**Comment by norbertschuler** [comment]
> Solved multiple donations when using Gifts + Native pay

Which criteria do I have to fulfill to test native payment? Currently I was not able to do test a native payment.

**Comment by sagararyal** [comment]
@norbertschuler for Apple Pay to work we need to add the domain on the stripe account. this is why Vercel urls do not support Apple Pay

For Google Pay or Payment Request button on chrome, no extra setup is needed besides adding card on chrome.

**Comment by harshvitra** [comment]
Are there any changes needed on this branch ?

**Comment by norbertschuler** [comment]
> Are there any changes needed on this branch ? 

Sorry, I still could not test. But it looks ok.

**Review by norbertschuler — APPROVED** [review]

**Pull request merged**